ktor-client/ktor-client-tests/jvm/src/io/ktor/client/tests/utils/ClientLoader.kt

```diff
@@ -29,10 +29,10 @@ actual abstract class ClientLoader {
      * Perform test against all clients from dependencies.
      */
     actual fun clientTests(
-        skipPlatforms: List<String>,
+        skipEngines: List<String>,
         block: suspend TestClientBuilder<HttpClientEngineConfig>.() -> Unit
     ) {
-        if ("jvm" in skipPlatforms) return
+        if (skipEngines.contains(engine.factory::class.simpleName)) return
```

Comment:
Consider using `"$engine"` instead of `engine.factory::class.simpleName`

Reply:
Done, I use `engine.toString()`.